src/main.rs

```diff
@@ -136,10 +138,23 @@ fn is_file_extension(file: &Path, extension: &str) -> bool {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use assert_cmd::Command as AssertCommand;
+    use clap::CommandFactory;
+    use std::fs::File;
 
     #[test]
     fn clap_cli_construction() {
-        use clap::CommandFactory;
         Cli::command().debug_assert();
     }
+
+    #[test]
+    fn pipe_stdin_to_cli() {
+        // TODO: create a file instead of reading one in the repo
+        let mut file = File::create("tests.txt").unwrap();
+        file.write_all(b"Hello, world!").unwrap();
+        let mut binding = AssertCommand::cargo_bin(env!("CARGO_PKG_NAME")).unwrap();
```

Comment:
> as said on Discord

i think you need to run `cargo build` for `$env.CARGO_PKG_NAME` to exist for real on the filesystem
```bash
cargo clean
cargo test
```
vs
```bash
cargo clean
cargo build
cargo test
```
=> the former fails and the latter runs successfully :ok_hand:

Reply:
Thanks!